qualtran/surface_code/msft_resource_estimator_interop.ipynb

```diff
@@ -154,7 +154,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.8"
```

Comment:
nit: remove this from the diff right?

Reply:
we usually let this churn in if there's an actual change to the notebook content (until we get better tooling in place)